int32 neck_x_rotation
int32 neck_z_rotation
int32 mouth
int32 upper_head
int32 left_eye
int32 right_eye
int32 flag
